### DISCUSSION TITLE
Anubis g6 transmitter, dexcom one sensor and Xdrip+

### DISCUSSION THREAD

**Discussion by Agio56** [NightscoutFoundation/xDrip#3098] [Q&A]
Is it possible to use Anubis g6 transmitter with Dexcom one sensors?
And also, is it compatible with Xdrip+?

**Comment by Navid200** [comment]
You can ask any questions you have about modified transmitters here: https://www.facebook.com/groups/anubisg6

**Reply by Agio56** [reply]
Thank you for the information. Unfortunately I don't have a Facebook account.

**Reply by Navid200** [reply]
The person you got the transmitter from has all the info.

**Comment by Nerienke** [comment]
Where can I buy the Anubis transmitter?

**Reply by Agio56** [reply]
On Facebook at the address you find in the messages above you can download a form to forward the request with instructions
